src/main/java/artifacts/mixin/mixins/client/render/PlayerEntityRendererMixin.java

```diff
@@ -44,13 +47,16 @@ private static GloveItem getGloveToRenderForHand(PlayerEntity player, Hand hand)
 			return null;
 		}
 
-		String slotGroup = hand == Hand.MAIN_HAND ? SlotGroups.HAND : SlotGroups.OFFHAND;
-		ItemStack stack = TrinketsApi.getTrinketComponent(player).getStack(slotGroup, Slots.GLOVES);
+		String slotGroup = hand == Hand.MAIN_HAND ? "hand" : "offhand";
+		Optional<TrinketComponent> maybeComponent = TrinketsApi.getTrinketComponent(player);
```

Comment:
I usually don't indicate in my variable names that they're optional with "maybe", so "comp" or "component" is fine here.